src/cookies.js

```diff
@@ -23,10 +23,35 @@ export default {
     const expires = 60 * 60 * 24 * 30;
     Cookies.set(EXPERIMENT_COOKIE_KEY, token, { expires, path: '/' });
   },
-  removePreviewCookie(path, domain) {
-    const pathOption = path ? { path } : {};
-    const domainOption = domain ? { domain } : {};
-    const options = Object.assign(pathOption, domainOption);
-    Cookies.remove(PREVIEW_COOKIE_KEY, options);
+  removePreviewCookie() {
+    demolishCookie(PREVIEW_COOKIE_KEY);
   },
 };
+
+
+function demolishCookie(name) {
```

Comment:
not really easy to understand and it doesn't seem to match all cases. just tried it with "www.google.fr"

Reply:
We should stick with the previous version since it works for a long time and it's clearer